src/server/game/Loot/LootMgr.cpp

```diff
@@ -408,6 +417,24 @@ bool LootItem::AllowedForPlayer(Player const* player) const
     if ((pProto->Flags2 & ITEM_FLAGS_EXTRA_ALLIANCE_ONLY) && player->GetTeamId(true) != TEAM_ALLIANCE)
         return false;
 
+    // Master looter can see certain items even if the character can't loot them
+    if (!isGivenByMasterLooter && isMasterLooter)
+    {
+        // check quest requirements
+        if (!(pProto->FlagsCu & ITEM_FLAGS_CU_IGNORE_QUEST_STATUS) && ((needs_quest || (pProto->StartQuest && player->GetQuestStatus(pProto->StartQuest) != QUEST_STATUS_NONE)) && !player->HasQuestForItem(itemid)))
+            return false;
+
+        return true;
```

Comment:
```suggestion
        return !(!(pProto->FlagsCu & ITEM_FLAGS_CU_IGNORE_QUEST_STATUS) && ((needs_quest || (pProto->StartQuest && player->GetQuestStatus(pProto->StartQuest) != QUEST_STATUS_NONE)) && !player->HasQuestForItem(itemid)))
```

Reply:
For readability reason, I think it's okay now.

Reply:
ok :+1: